astropy/wcs/src/unit_list_proxy.c

```diff
@@ -13,6 +13,7 @@
  ***************************************************************************/
 
 #define MAXSIZE 68
+#define ARRAYSIZE 72
```

Comment:
This should probably also be used everywhere we currently have hardcoded the 72.